Status.md

```diff
@@ -0,0 +1,48 @@
+# KOB Email Verification Service
+##  About Project
+ * Proves the ownership of user over his email.
+ * Provides verified digital credential for user’s email which is stored in user’s wallet.
+ * User submit this digital credential, when he provides his email for availing third party service.
+ *  Eg :OTP verification during form filling or transaction.
+ * Prerequisites: Git, GitHub, python, Django, HTML5, Docker,Docker-compose.
```

Comment:
Prerequisites could be a seperate section and it only has to mention what should a user have/install before trying out the service.